modules/my_health/app/helpers/my_health/prescription_helper.rb

```diff
@@ -28,6 +28,9 @@ def filter_data_by_refill_and_renew(data)
       end
 
       def renewable(item)
+        # UHD prescriptions don't have disp_status, return false for them
```

Comment:
should we just added a nulled disp_status to the UHD service so you don't need to do this here?